apps/src/templates/DropdownButton.js

```diff
@@ -124,7 +125,11 @@ export const DropdownButton = class DropdownButtonComponent extends Component {
           iconStyle={styles.icon}
           color={color}
           className={this.props.className}
-        />
+        >
+          {this.props.customText && (
+            <div style={styles.main}>{this.props.customText}</div>
```

Comment:
I don't see the styles.main added here, was that intentional?

Reply:
(recording offline conversation) yes- this style just has one line: display: 'inline-block'. Instead of making a duplicate styling because this was all I needed, I reused this style.